modules/gapi/test/gapi_gcompiled_tests.cpp

```diff
@@ -170,4 +182,10 @@ TEST_F(GCompiledValidateMetaUntyped, InvalidMetaNumber)
     EXPECT_THROW(f(cv::gin(in1, sc), cv::gout(out1, out2)), std::logic_error);
 }
 
+TEST_F(GCompiledValidateMetaEmpty, InvalidMatMeta)
+{
+    EXPECT_THROW(m_ucc.compile(cv::empty_gmat_desc(),
+                               cv::empty_scalar_desc()),
+                 std::logic_error);
+}
```

Comment:
I believe these tests only check the case when we compile graph with the empty meta.

The reproducer in the original ticket is a little bit different

Reply:
additional tests added